src/main/java/nextstep/domain/Line.java

```diff
@@ -1,17 +1,21 @@
 package nextstep.domain;
 
-import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
 import java.util.stream.IntStream;
 
 public class Line {
 
-    private static final RandomTransverseBarStrategy DEFAULT_TRANSVERSE_BAR_STRATEGY = new RandomTransverseBarStrategy();
+    private static final TransverseBarStrategy DEFAULT_TRANSVERSE_BAR_STRATEGY = new RandomTransverseBarStrategy();
 
     private List<Boolean> points;
```

Comment:
내부 구조를 잘 설계 해주셨기에 생성 이후로 값의 재 초기화가 될 일이 없는 것 같아요 👍 
해당 인스턴스 필드에 final 키워드까지 추가하면 좋을 것 같아요!
외부에서 임의로 내부의 값을 제어할 수 없기 때문에 객체의 자율성이 보장되고 프로그램 내에서 변하지 않고
고정된 부분이 많아짐으로써 프로그램 안정도를 높일 수 있을 것 같아요!

```suggestion
    private final List<Boolean> points;
```